scripts/main.js

```diff
@@ -1 +1,38 @@
 console.log("[main.js] Test main.js");
+setTimeout(function () {
+
+    /* Image Slider - Swiper */
+    var imageSlider = new Swiper(".image-slider", {
```

Comment:
Avoid using `var`. Prefer `const` (or `let` if you need to reassign its value).

```suggestion
    const imageSlider = new Swiper(".image-slider", {
```